routing/cross_mwm_graph.cpp

```diff
@@ -0,0 +1,222 @@
+#include "routing/cross_mwm_graph.hpp"
+
+#include "indexer/scales.hpp"
+
+#include "geometry/mercator.hpp"
+
+#include "base/stl_helpers.hpp"
+
+#include "defines.hpp"
+
+#include <algorithm>
+#include <numeric>
+#include <utility>
+
+using namespace routing;
+using namespace std;
+
+namespace
+{
+double constexpr kTransitionEqualityDistM = 20.0;
```

Comment:
Now it's not implemented and cannot be used. When it's done I'll remove this code and use to for case of transition osm id - osm id. All other transition case (getting twins): osm id - osrm, osrm - osm id and osrm - osrm will be implemented based on coords.